### DISCUSSION TITLE
FR: Support for Vladmantic and Anapnoe WebUI

### DISCUSSION THREAD

**Discussion by oddomatik** [AbdBarho/stable-diffusion-webui-docker#422] [Ideas]
Apparently Vladmantic and Anapnoe are collaborating to make a really nice webui. I haven't personally used either, but they both seem to already have worthy contributions in their respective areas of expertise that would make good additions to this docker image. I may explore some code contributions, but that is a low priority for me right now. I mainly just wanted to make this space aware of where things might be going in case they are projects worth keeping an eye on to integrate here.

**Comment by AbdBarho** [comment]
Yeah, see https://github.com/AbdBarho/stable-diffusion-webui-docker/discussions/407

I generally think it is great that the community is always getting bigger and we have more UIs. Unfortunately, it is more maintenance overhead on my side so I tend to procrastinate when it comes to adding new UIs, especially ones that I have not used before.

Anyway, contributions are always welcome!

**Reply by oddomatik** [reply]
Very good, thanks for the response! I'm sorry I missed that discussion in my brief search. If I do get around to trying it I'll do so in a sustainable and contributive way.
